packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    @include carbon--make-row;
+
+    &__heading {
+      margin-bottom: $carbon--layout-03;
```

Comment:
What specs were you referring to? The ones I'm looking at says `64px`, not `32px`. So that would be `layout-05` instead.

![image](https://user-images.githubusercontent.com/3793636/74486864-a12d3c00-4e83-11ea-8f23-068a1d59bed0.png)